webapp/shop/cube/views.py

```diff
@@ -34,6 +34,10 @@ def cube_microcerts(
     """
     View for Microcerts homepage
     """
+
+    # Temporarily redirect to /cube while the beta is closed
+    return flask.redirect("/cube", code=302)
```

Comment:
I think you could just use `redirects.yaml` for this stuff. The check is always done before each request is handled so I think you could actually just leave the views as they are.

Reply:
Thanks, I wasn't aware of `redirects.yaml`. I've made the change and it seems to work fine.